CMakePresets.json

```diff
@@ -33,15 +36,17 @@
       "inherits": "defaults",
       "generator": "Ninja Multi-Config",
       "cacheVariables": {
-        "CMAKE_CONFIGURATION_TYPES": "Debug;Release;RelWithDebInfo;MinSizeRel",
-        "PREVIEW_BUILD_TESTS": "Off"
+        "CMAKE_CONFIGURATION_TYPES": "Debug;Release;RelWithDebInfo;MinSizeRel"
       }
     },
     {
       "name": "stm32",
       "hidden": true,
       "inherits": "defaults",
       "generator": "Ninja Multi-Config",
+      "environment": {
+        "CMAKE_TOOLCHAIN_FILE": ""
```

Comment:
I think this is one of the few places where a comment would be acceptable. I imagine one could easily see this as a redundant statement, and remove it.

Reply:
I added "explain_CMAKE_TOOLCHAIN_FILE": "Needed as workaround for not requiring the toolchain before fetching EmIL" right before that variable